src/ONYXKEYS.js

```diff
@@ -30,6 +30,10 @@ export default {
     // Contains all the personalDetails the user has access to
     PERSONAL_DETAILS: 'personalDetails',
 
+    // Saves the current country code by IP which is displayed when the user types a phone number without
+    // international code
+    COUNTRY_CODE_BY_IP: 'countryCodeByIP',
```

Comment:
Sure, i soft disagee. Atm i named it what it [is](https://github.com/Expensify/Web-Expensify/blob/2d17b9582b720e2480e4ff6fe59d3802f1aa17e3/lib/Geo/LocationUtils.php#L52). 

> Maybe one day the user is able to set their country

Planning for that sounds like preoptimization and when that time comes then we can change the const. Thoughts?

Also as i write this out is there a way to clear unused ONYX saved data from the users local storage?

Reply:
Well if you are soft on this can you please change it? If we can name something correctly and consistently on the first pass I say let's do it. All of the other keys in this file describe the data they do not tell us anything about where it came from. If for no other reason do it for consistency and so the next person who saves something doesn't follow this example.